services/galley/test/integration/API.hs

```diff
@@ -389,62 +390,76 @@ postConvWithRemoteUsersOk rbs = do
       assertConv
         rsp
         RegularConv
         alice
         qAlice
-        shouldBePresent
-        (Just nameMaxSize)
+        (otherLocals <> participatingRemotes)
+        (Just convName)
         Nothing
     let cid = qUnqualified qcid
     cvs <- mapM (convView qcid) [alice, alex, amy]
     liftIO $
       mapM_ WS.assertSuccess
         =<< Async.mapConcurrently (checkWs qAlice) (zip cvs [wsAlice, wsAlex, wsAmy])
 
-    liftIO $
-      -- as many federated requests as remote backends, i.e., one per remote backend
-      Set.size (Set.fromList $ frTargetDomain <$> federatedRequests) @?= Set.size rbs
-    let fedReq = head federatedRequests
-    fedReqBody <- assertRight $ parseFedRequest fedReq
-
     liftIO $ do
-      length federatedRequests @?= Set.size rbs
-
-      F.ccOrigUserId fedReqBody @?= alice
-      F.ccCnvId fedReqBody @?= cid
-      F.ccCnvType fedReqBody @?= RegularConv
-      F.ccCnvAccess fedReqBody @?= [InviteAccess]
-      F.ccCnvAccessRoles fedReqBody
+      let expectedReqs =
+            Set.fromList $
+              [ "on-conversation-created",
+                "on-conversation-updated"
+              ]
+       in assertBool "Some federated calls are missing" $
+            expectedReqs `Set.isSubsetOf` Set.fromList (frRPC <$> federatedRequests)
+
+    let fedReqsCreated = filter (\r -> frRPC r == "on-conversation-created") federatedRequests
+    fedReqCreatedBodies <- for fedReqsCreated $ assertRight . parseFedRequest
+    forM_ fedReqCreatedBodies $ \fedReqCreatedBody -> liftIO $ do
+      F.ccOrigUserId fedReqCreatedBody @?= alice
+      F.ccCnvId fedReqCreatedBody @?= cid
+      F.ccCnvType fedReqCreatedBody @?= RegularConv
+      F.ccCnvAccess fedReqCreatedBody @?= [InviteAccess]
+      F.ccCnvAccessRoles fedReqCreatedBody
         @?= Set.fromList [TeamMemberAccessRole, NonTeamMemberAccessRole, ServiceAccessRole]
-      F.ccCnvName fedReqBody @?= Just nameMaxSize
+      F.ccCnvName fedReqCreatedBody @?= Just convName
       assertBool "Notifying an incorrect set of conversation members" $
-        shouldBePresentSet `Set.isSubsetOf` F.ccNonCreatorMembers fedReqBody
-      F.ccMessageTimer fedReqBody @?= Nothing
-      F.ccReceiptMode fedReqBody @?= Nothing
-
-      fedBodies <- forM federatedRequests $ assertRight . parseFedRequest
-      assertBool "Not all request bodies are equal" (all (fedReqBody ==) fedBodies)
+        minimalShouldBePresentSet `Set.isSubsetOf` F.ccNonCreatorMembers fedReqCreatedBody
+      F.ccMessageTimer fedReqCreatedBody @?= Nothing
+      F.ccReceiptMode fedReqCreatedBody @?= Nothing
+
+    let fedReqsAdd = filter (\r -> frRPC r == "on-conversation-updated") federatedRequests
+    fedReqAddBodies <- for fedReqsAdd $ assertRight . parseFedRequest
+    forM_ fedReqAddBodies $ \fedReqAddBody -> liftIO $ do
+      F.cuOrigUserId fedReqAddBody @?= qAlice
+      F.cuConvId fedReqAddBody @?= cid
+      assertBool "The list of already present users should be non-empty"
```

Comment:
```suggestion
      -- This remote backend must already have their users in the conversation, otherwise they should not be receiving the conversation update message
      assertBool "The list of already present users should be non-empty"
```